### PR TITLE
ELPDES-129: Range Slider

### PR DESCRIPTION
**Pull request by pauzim**
Issue not resolved: Regardless of using `-moz` vendor prefixes, slider isn't working in Mozilla 👽
Bug added: https://elpassion.atlassian.net/jira/software/projects/ELPDES/boards/246?selectedIssue=ELPDES-178

### CODE REVIEW

**Comment by piotrekzawada** [comment]
@pauzim is it only layout component or should it be working? It doesn't work for me in either chrome or firefox 😢

**Comment by piotrekzawada** [comment]
firefox: 
image

**Comment by pauzim** [comment]
> @pauzim is it only layout component or should it be working? It doesn't work for me in either chrome or firefox 😢

@piotrekzawada 😱 should be working - tested in chrome and safari. I'll check what's happened

File: packages/design-system/src/stories/atoms/RangeSlider.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+  return (
+    <div className="ml-4">
+      <RangeSliderComponent {...props} onChange={setValue} value={value} />
+    </div>
+  );
+};
+
+export default {
+  title: "🟢 Atoms/RangeSlider",
```
Comment (mzyznowska) [inline packages/design-system/src/stories/atoms/RangeSlider.stories.tsx:19]: It should be an orange ball because this component wasn't tested yet.

**Review by mzyznowska — DISMISSED** [review]

**Review by mzyznowska — COMMENTED** [review]
I approved but I checked if it works after I saw Piotrek's comment

**Comment by pauzim** [comment]
@piotrekzawada , @mzyznowska - Small fixes added, please take a 👀 if it's working now 🙏

**Review by piotrekzawada — APPROVED** [review]

**Pull request merged**